README.md

```diff
@@ -1,4 +1,18 @@
 Fences
 ======
 
-A module adding fence blocks
+This module adds a simple feature, a fence block.
+
+Each fence block can be connected to up to four neighbouring blocks. Currently, however, the module is a little bit bugged. This means that the fences do not connect correctly the very moment they are placed. The issue can be fixed by forcing an update for given fence blocks, for instance by placing and mining arbitrary block right next to the fence. Another problem this module currently has is that the ground right around fence is severely darkened if you have SSAO enabled in your graphics settings. This module also lacks any sort of gate, although it used to have it in past.
```

Comment:
Change the second sentence please. It sounds very unprofessional. Something like "This module is still under development, and has a few bugs" would be much better

Reply:
Also this para should be **after** the Getting Started. Never talk about downfalls at the start.